diff --git BDInfo/BDInfoLib.csproj BDInfo/BDInfoLib.csproj
index 55d3711..47870de 100644
--- BDInfo/BDInfoLib.csproj
+++ BDInfo/BDInfoLib.csproj
@@ -5,7 +5,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{3209AF1D-4762-4960-9DE5-13E90E77EE96}</ProjectGuid>
+    <ProjectGuid>{CF10DEBE-9A1B-4CD6-B566-E6013C1E6CB9}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>BDInfoLib</RootNamespace>
diff --git Caliburn.Metro.Core/Caliburn.Metro.Core.csproj Caliburn.Metro.Core/Caliburn.Metro.Core.csproj
index f164f54..8caa82a 100644
--- Caliburn.Metro.Core/Caliburn.Metro.Core.csproj
+++ Caliburn.Metro.Core/Caliburn.Metro.Core.csproj
@@ -32,22 +32,11 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Caliburn.Micro, Version=1.5.2.0, Culture=neutral, PublicKeyToken=8e5891231f2ed21f, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Caliburn.Micro.1.5.2\lib\net45\Caliburn.Micro.dll</HintPath>
-    </Reference>
-    <Reference Include="MahApps.Metro, Version=0.10.0.0, Culture=neutral, PublicKeyToken=f4fb5a3c4d1e5b4f, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\MahApps.Metro.0.10.1.1\lib\net45\MahApps.Metro.dll</HintPath>
-    </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core" />
-    <Reference Include="System.Windows.Interactivity, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\MahApps.Metro.0.10.1.1\lib\net45\System.Windows.Interactivity.dll</HintPath>
-    </Reference>
     <Reference Include="System.Xaml" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Data.DataSetExtensions" />
@@ -61,10 +50,6 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
-    <None Include="Resources\Entypo.ttf" />
-  </ItemGroup>
-  <ItemGroup>
     <Page Include="Resources\Icons.xaml">
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
@@ -73,6 +58,16 @@
   <ItemGroup>
     <Content Include="Resources\Entypo-license.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Caliburn.Micro\src\Caliburn.Micro.Platform\Caliburn.Micro.Platform.NET45.csproj">
+      <Project>{ef2c0c7d-3ffe-4884-9aac-7622fee66809}</Project>
+      <Name>Caliburn.Micro.Platform.NET45</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\MahApps.Metro\MahApps.Metro\MahApps.Metro.NET45.csproj">
+      <Project>{942d11c4-29f5-46c4-8c14-4b976d56c637}</Project>
+      <Name>MahApps.Metro.NET45</Name>
+    </ProjectReference>
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\nuget.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
diff --git Caliburn.Metro.Core/Resources/Entypo.ttf Caliburn.Metro.Core/Resources/Entypo.ttf
deleted file mode 100644
index d6d7687..0000000
Binary files Caliburn.Metro.Core/Resources/Entypo.ttf and /dev/null differ
diff --git Caliburn.Metro.Core/packages.config Caliburn.Metro.Core/packages.config
deleted file mode 100644
index 3907eef..0000000
--- Caliburn.Metro.Core/packages.config
+++ /dev/null
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Caliburn.Micro" version="1.5.2" targetFramework="net45" />
-  <package id="MahApps.Metro" version="0.10.1.1" targetFramework="net45" />
-  <package id="MahApps.Metro.Resources" version="0.1.0.1" targetFramework="net45" />
-</packages>
\ No newline at end of file
diff --git Caliburn.Metro/Caliburn.Metro.csproj Caliburn.Metro/Caliburn.Metro.csproj
index fa1c871..65dd65f 100644
--- Caliburn.Metro/Caliburn.Metro.csproj
+++ Caliburn.Metro/Caliburn.Metro.csproj
@@ -32,22 +32,11 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Caliburn.Micro, Version=1.5.2.0, Culture=neutral, PublicKeyToken=8e5891231f2ed21f, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Caliburn.Micro.1.5.2\lib\net45\Caliburn.Micro.dll</HintPath>
-    </Reference>
-    <Reference Include="MahApps.Metro, Version=0.10.0.0, Culture=neutral, PublicKeyToken=f4fb5a3c4d1e5b4f, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\MahApps.Metro.0.10.1.1\lib\net45\MahApps.Metro.dll</HintPath>
-    </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core" />
-    <Reference Include="System.Windows.Interactivity, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\MahApps.Metro.0.10.1.1\lib\net45\System.Windows.Interactivity.dll</HintPath>
-    </Reference>
     <Reference Include="System.Xaml" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Data.DataSetExtensions" />
@@ -61,12 +50,6 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="Resources\Entypo.ttf" />
-  </ItemGroup>
-  <ItemGroup>
     <Page Include="Resources\Icons.xaml">
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
@@ -76,6 +59,14 @@
     <Content Include="Resources\Entypo-license.txt" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\Caliburn.Micro\src\Caliburn.Micro.Platform\Caliburn.Micro.Platform.NET45.csproj">
+      <Project>{ef2c0c7d-3ffe-4884-9aac-7622fee66809}</Project>
+      <Name>Caliburn.Micro.Platform.NET45</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\MahApps.Metro\MahApps.Metro\MahApps.Metro.NET45.csproj">
+      <Project>{942d11c4-29f5-46c4-8c14-4b976d56c637}</Project>
+      <Name>MahApps.Metro.NET45</Name>
+    </ProjectReference>
     <ProjectReference Include="..\Caliburn.Metro.Core\Caliburn.Metro.Core.csproj">
       <Project>{3e257c20-b95a-40c5-881e-9f3d834ccb4e}</Project>
       <Name>Caliburn.Metro.Core</Name>
diff --git Caliburn.Metro/Resources/Entypo.ttf Caliburn.Metro/Resources/Entypo.ttf
deleted file mode 100644
index d6d7687..0000000
Binary files Caliburn.Metro/Resources/Entypo.ttf and /dev/null differ
diff --git Caliburn.Metro/packages.config Caliburn.Metro/packages.config
deleted file mode 100644
index 3907eef..0000000
--- Caliburn.Metro/packages.config
+++ /dev/null
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Caliburn.Micro" version="1.5.2" targetFramework="net45" />
-  <package id="MahApps.Metro" version="0.10.1.1" targetFramework="net45" />
-  <package id="MahApps.Metro.Resources" version="0.1.0.1" targetFramework="net45" />
-</packages>
\ No newline at end of file
diff --git src/Castle.Windsor/Castle.Windsor.csproj src/Castle.Windsor/Castle.Windsor.csproj
index aa58ab2..1101864 100644
--- src/Castle.Windsor/Castle.Windsor.csproj
+++ src/Castle.Windsor/Castle.Windsor.csproj
@@ -2,6 +2,7 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <RootPath Condition=" '$(RootPath)' == '' ">$(MSBuildProjectDirectory)\..\..</RootPath>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup>
     <ProjectType>Local</ProjectType>
@@ -43,6 +44,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET40-Debug|AnyCPU'">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
@@ -65,6 +67,7 @@
     <CodeAnalysisIgnoreBuiltInRuleSets>false</CodeAnalysisIgnoreBuiltInRuleSets>
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET45-Release|AnyCPU'">
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
@@ -77,6 +80,7 @@
     <DebugType>pdbonly</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET45-Debug|AnyCPU'">
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
@@ -89,6 +93,7 @@
     <DebugType>full</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET40CP-Release|AnyCPU'">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
@@ -111,6 +116,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET40CP-Debug|AnyCPU'">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
@@ -132,6 +138,7 @@
     <CodeAnalysisIgnoreBuiltInRuleSets>false</CodeAnalysisIgnoreBuiltInRuleSets>
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET35-Release|AnyCPU'">
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
@@ -155,6 +162,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'NET35-Debug|AnyCPU'">
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
@@ -177,6 +185,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SL4-Release|AnyCPU'">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
@@ -191,6 +200,7 @@
     <DebugType>pdbonly</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SL4-Debug|AnyCPU'">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
@@ -205,6 +215,7 @@
     <DebugType>full</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SL5-Release|AnyCPU'">
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
@@ -217,6 +228,7 @@
     <DebugType>pdbonly</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SL5-Debug|AnyCPU'">
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
@@ -229,6 +241,7 @@
     <DebugType>full</DebugType>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'MONO26-Debug|AnyCPU'">
     <TargetFrameworkProfile>
@@ -251,6 +264,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'MONO26-Release|AnyCPU'">
     <TargetFrameworkProfile>
@@ -273,6 +287,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'MONO28-Debug|AnyCPU'">
     <TargetFrameworkProfile>
@@ -295,6 +310,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>false</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'MONO28-Release|AnyCPU'">
     <TargetFrameworkProfile>
@@ -317,6 +333,7 @@
     <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
     <CodeAnalysisFailOnMissingRules>false</CodeAnalysisFailOnMissingRules>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup Condition="$(TargetFrameworkProfile)!='Client' and $(MSBuildTargets)!='Silverlight 4.0' and $(MSBuildTargets)!='Silverlight 5.0'">
     <Reference Include="System.Web" />
@@ -328,16 +345,10 @@
       <Name>System.Data</Name>
     </Reference>
   </ItemGroup>
-  <ItemGroup Condition="$(MSBuildTargets)=='Silverlight 4.0' or $(MSBuildTargets)=='Silverlight 5.0'">
-    <Reference Include="System.Windows" />
-  </ItemGroup>
   <ItemGroup>
-    <Reference Include="Castle.Core">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\lib\$(BuildConfigKey)\Castle.Core.dll</HintPath>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Windows" />
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
@@ -1012,6 +1023,12 @@
     <Folder Include="MicroKernel\SubSystems\Scoping\" />
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Castle.Core\src\Castle.Core\Castle.Core.csproj">
+      <Project>{e4fa5b53-7d36-429e-8e5c-53d5479242ba}</Project>
+      <Name>Castle.Core</Name>
+    </ProjectReference>
+  </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" Condition="($(MSBuildTargets) == '') Or ($(MSBuildTargets) == 'CSharp')" />
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\Silverlight\v4.0\Microsoft.Silverlight.CSharp.targets" Condition="$(MSBuildTargets) == 'Silverlight 4.0'" />
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\Silverlight\v5.0\Microsoft.Silverlight.CSharp.targets" Condition="$(MSBuildTargets) == 'Silverlight 5.0'" />
diff --git src/DirectShowLib-2012.csproj src/DirectShowLib-2012.csproj
index c722ec4..5a1b63f 100644
--- src/DirectShowLib-2012.csproj
+++ src/DirectShowLib-2012.csproj
@@ -1,217 +1,217 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
-  <PropertyGroup>
-    <ProjectType>Local</ProjectType>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{CE009BFA-5EF6-4153-B2A1-2FA79698A9AE}</ProjectGuid>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ApplicationIcon>
-    </ApplicationIcon>
-    <AssemblyKeyContainerName>
-    </AssemblyKeyContainerName>
-    <AssemblyName>DirectShowLib-2005</AssemblyName>
-    <AssemblyOriginatorKeyFile>DShowNET.snk</AssemblyOriginatorKeyFile>
-    <DefaultClientScript>JScript</DefaultClientScript>
-    <DefaultHTMLPageLayout>Grid</DefaultHTMLPageLayout>
-    <DefaultTargetSchema>IE50</DefaultTargetSchema>
-    <DelaySign>false</DelaySign>
-    <OutputType>Library</OutputType>
-    <RootNamespace>DirectShowLib</RootNamespace>
-    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
-    <StartupObject>
-    </StartupObject>
-    <FileUpgradeFlags>
-    </FileUpgradeFlags>
-    <UpgradeBackupLocation>
-    </UpgradeBackupLocation>
-    <SignAssembly>true</SignAssembly>
-    <OldToolsVersion>3.5</OldToolsVersion>
-    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <OutputPath>bin\Debug\</OutputPath>
-    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
-    <BaseAddress>285212672</BaseAddress>
-    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
-    <ConfigurationOverrideFile>
-    </ConfigurationOverrideFile>
-    <DefineConstants>TRACE;DEBUG;ALLOW_UNTESTED_STRUCTS;ALLOW_UNTESTED_INTERFACES;USING_NET20</DefineConstants>
-    <DocumentationFile>
-    </DocumentationFile>
-    <DebugSymbols>true</DebugSymbols>
-    <FileAlignment>4096</FileAlignment>
-    <NoStdLib>false</NoStdLib>
-    <NoWarn>
-    </NoWarn>
-    <Optimize>false</Optimize>
-    <RegisterForComInterop>false</RegisterForComInterop>
-    <RemoveIntegerChecks>false</RemoveIntegerChecks>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <WarningLevel>4</WarningLevel>
-    <DebugType>full</DebugType>
-    <ErrorReport>prompt</ErrorReport>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <OutputPath>bin\Release\</OutputPath>
-    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
-    <BaseAddress>285212672</BaseAddress>
-    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
-    <ConfigurationOverrideFile>
-    </ConfigurationOverrideFile>
-    <DefineConstants>USING_NET20</DefineConstants>
-    <DocumentationFile>
-    </DocumentationFile>
-    <DebugSymbols>false</DebugSymbols>
-    <FileAlignment>4096</FileAlignment>
-    <NoStdLib>false</NoStdLib>
-    <NoWarn>
-    </NoWarn>
-    <Optimize>true</Optimize>
-    <RegisterForComInterop>false</RegisterForComInterop>
-    <RemoveIntegerChecks>false</RemoveIntegerChecks>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <WarningLevel>4</WarningLevel>
-    <DebugType>none</DebugType>
-    <ErrorReport>prompt</ErrorReport>
-    <UseVSHostingProcess>true</UseVSHostingProcess>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Drawing" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="amstream.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="amvideo.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="AssemblyInfo.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="atscpsipparser.cs" />
-    <Compile Include="austream.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="AxCore.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="AXExtend.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="BDAIface.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Bdatif.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="BDATypes.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Control.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="DES.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="DevEnum.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="DMODShow.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="dsattrib.cs" />
-    <Compile Include="DsUtils.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="dvbsiparser.cs" />
-    <Compile Include="DVDIf.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="DynGraph.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="encdec.cs" />
-    <Compile Include="EvCode.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="IL21Dec.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="MediaObj.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="MediaParam.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Misc.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Mixerocx.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="mmstream.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="mpconfig.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Mpeg2Data.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="mpeg2psiparser.cs" />
-    <Compile Include="Mpeg2Structs.cs" />
-    <Compile Include="other.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="PlayList.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="QEdit.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="qnetwork.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="RegBag.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Sbe.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Tuner.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="TvRatings.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Uuids.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Vidcap.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="VidPort.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="Vmr9.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="VmRender.cs">
-      <SubType>Code</SubType>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="app.config" />
-    <None Include="DShowNET.snk" />
-  </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PreBuildEvent>
-    </PreBuildEvent>
-    <PostBuildEvent>
-    </PostBuildEvent>
-  </PropertyGroup>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <PropertyGroup>
+    <ProjectType>Local</ProjectType>
+    <ProductVersion>9.0.30729</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{D623A98B-9DDD-4035-8495-FAF4C641046D}</ProjectGuid>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ApplicationIcon>
+    </ApplicationIcon>
+    <AssemblyKeyContainerName>
+    </AssemblyKeyContainerName>
+    <AssemblyName>DirectShowLib-2005</AssemblyName>
+    <AssemblyOriginatorKeyFile>DShowNET.snk</AssemblyOriginatorKeyFile>
+    <DefaultClientScript>JScript</DefaultClientScript>
+    <DefaultHTMLPageLayout>Grid</DefaultHTMLPageLayout>
+    <DefaultTargetSchema>IE50</DefaultTargetSchema>
+    <DelaySign>false</DelaySign>
+    <OutputType>Library</OutputType>
+    <RootNamespace>DirectShowLib</RootNamespace>
+    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
+    <StartupObject>
+    </StartupObject>
+    <FileUpgradeFlags>
+    </FileUpgradeFlags>
+    <UpgradeBackupLocation>
+    </UpgradeBackupLocation>
+    <SignAssembly>true</SignAssembly>
+    <OldToolsVersion>3.5</OldToolsVersion>
+    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <OutputPath>bin\Debug\</OutputPath>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+    <BaseAddress>285212672</BaseAddress>
+    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
+    <ConfigurationOverrideFile>
+    </ConfigurationOverrideFile>
+    <DefineConstants>TRACE;DEBUG;ALLOW_UNTESTED_STRUCTS;ALLOW_UNTESTED_INTERFACES;USING_NET20</DefineConstants>
+    <DocumentationFile>
+    </DocumentationFile>
+    <DebugSymbols>true</DebugSymbols>
+    <FileAlignment>4096</FileAlignment>
+    <NoStdLib>false</NoStdLib>
+    <NoWarn>
+    </NoWarn>
+    <Optimize>false</Optimize>
+    <RegisterForComInterop>false</RegisterForComInterop>
+    <RemoveIntegerChecks>false</RemoveIntegerChecks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningLevel>4</WarningLevel>
+    <DebugType>full</DebugType>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <OutputPath>bin\Release\</OutputPath>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+    <BaseAddress>285212672</BaseAddress>
+    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
+    <ConfigurationOverrideFile>
+    </ConfigurationOverrideFile>
+    <DefineConstants>USING_NET20</DefineConstants>
+    <DocumentationFile>
+    </DocumentationFile>
+    <DebugSymbols>false</DebugSymbols>
+    <FileAlignment>4096</FileAlignment>
+    <NoStdLib>false</NoStdLib>
+    <NoWarn>
+    </NoWarn>
+    <Optimize>true</Optimize>
+    <RegisterForComInterop>false</RegisterForComInterop>
+    <RemoveIntegerChecks>false</RemoveIntegerChecks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningLevel>4</WarningLevel>
+    <DebugType>none</DebugType>
+    <ErrorReport>prompt</ErrorReport>
+    <UseVSHostingProcess>true</UseVSHostingProcess>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Drawing" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="amstream.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="amvideo.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="AssemblyInfo.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="atscpsipparser.cs" />
+    <Compile Include="austream.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="AxCore.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="AXExtend.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="BDAIface.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Bdatif.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="BDATypes.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Control.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="DES.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="DevEnum.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="DMODShow.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="dsattrib.cs" />
+    <Compile Include="DsUtils.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="dvbsiparser.cs" />
+    <Compile Include="DVDIf.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="DynGraph.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="encdec.cs" />
+    <Compile Include="EvCode.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="IL21Dec.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="MediaObj.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="MediaParam.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Misc.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Mixerocx.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="mmstream.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="mpconfig.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Mpeg2Data.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="mpeg2psiparser.cs" />
+    <Compile Include="Mpeg2Structs.cs" />
+    <Compile Include="other.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="PlayList.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="QEdit.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="qnetwork.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="RegBag.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Sbe.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Tuner.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="TvRatings.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Uuids.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Vidcap.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="VidPort.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="Vmr9.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Include="VmRender.cs">
+      <SubType>Code</SubType>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.config" />
+    <None Include="DShowNET.snk" />
+  </ItemGroup>
+  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PreBuildEvent>
+    </PreBuildEvent>
+    <PostBuildEvent>
+    </PostBuildEvent>
+  </PropertyGroup>
 </Project>
\ No newline at end of file
diff --git SevenZip/SevenZip2010.csproj SevenZip/SevenZip2010.csproj
index 6d4df12..be23438 100644
--- SevenZip/SevenZip2010.csproj
+++ SevenZip/SevenZip2010.csproj
@@ -12,7 +12,7 @@
     <AssemblyName>SevenZipSharp</AssemblyName>
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>SevenZip.snk</AssemblyOriginatorKeyFile>
     <FileUpgradeFlags>
     </FileUpgradeFlags>
@@ -47,6 +47,7 @@
     <RegisterForComInterop>false</RegisterForComInterop>
     <DocumentationFile>bin\Debug\SevenZipSharp.xml</DocumentationFile>
     <RunCodeAnalysis>true</RunCodeAnalysis>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
@@ -57,6 +58,7 @@
     <WarningLevel>4</WarningLevel>
     <RegisterForComInterop>false</RegisterForComInterop>
     <DocumentationFile>bin\Release\SevenZipSharp.xml</DocumentationFile>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Deploy|AnyCPU' ">
     <OutputPath>bin\Deploy\</OutputPath>
@@ -69,6 +71,7 @@
     <CodeAnalysisUseTypeNameInSuppression>true</CodeAnalysisUseTypeNameInSuppression>
     <CodeAnalysisModuleSuppressionsFile>GlobalSuppressions.cs</CodeAnalysisModuleSuppressionsFile>
     <ErrorReport>prompt</ErrorReport>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
diff --git SharpDvdInfo.csproj SharpDvdInfo.csproj
index f262b50..5fc05e3 100644
--- SharpDvdInfo.csproj
+++ SharpDvdInfo.csproj
@@ -1,68 +1,68 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{B713C29E-0E52-4B04-B4AD-AF714449A79C}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>SharpDvdInfo</RootNamespace>
-    <AssemblyName>SharpDvdInfo</AssemblyName>
-    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DocumentationFile>bin\Debug\SharpDvdInfo.xml</DocumentationFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="DvdInfoContainer.cs" />
-    <Compile Include="DvdTypes\DvdAudioFormat.cs" />
-    <Compile Include="DvdTypes\DvdAudioId.cs" />
-    <Compile Include="DvdTypes\DvdAudioQuantization.cs" />
-    <Compile Include="DvdTypes\DvdAudioType.cs" />
-    <Compile Include="DvdTypes\DvdLangList.cs" />
-    <Compile Include="DvdTypes\DvdLanguage.cs" />
-    <Compile Include="DvdTypes\DvdSubpictureFormat.cs" />
-    <Compile Include="DvdTypes\DvdSubpictureType.cs" />
-    <Compile Include="DvdTypes\DvdVideoAspectRatio.cs" />
-    <Compile Include="DvdTypes\DvdVideoMpegVersion.cs" />
-    <Compile Include="DvdTypes\DvdVideoPermittedDisplayFormat.cs" />
-    <Compile Include="DvdTypes\DvdVideoResolution.cs" />
-    <Compile Include="DvdTypes\DvdVideoStandard.cs" />
-    <Compile Include="Model\AudioProperties.cs" />
-    <Compile Include="Model\SubpictureProperties.cs" />
-    <Compile Include="Model\TitleInfo.cs" />
-    <Compile Include="Model\VideoProperties.cs" />
-    <Compile Include="Model\VmgmInfo.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProductVersion>8.0.30703</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{CBD3F86C-CF61-442D-9A71-FD092E571FFA}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>SharpDvdInfo</RootNamespace>
+    <AssemblyName>SharpDvdInfo</AssemblyName>
+    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <TargetFrameworkProfile />
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <DocumentationFile>bin\Debug\SharpDvdInfo.xml</DocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="DvdInfoContainer.cs" />
+    <Compile Include="DvdTypes\DvdAudioFormat.cs" />
+    <Compile Include="DvdTypes\DvdAudioId.cs" />
+    <Compile Include="DvdTypes\DvdAudioQuantization.cs" />
+    <Compile Include="DvdTypes\DvdAudioType.cs" />
+    <Compile Include="DvdTypes\DvdLangList.cs" />
+    <Compile Include="DvdTypes\DvdLanguage.cs" />
+    <Compile Include="DvdTypes\DvdSubpictureFormat.cs" />
+    <Compile Include="DvdTypes\DvdSubpictureType.cs" />
+    <Compile Include="DvdTypes\DvdVideoAspectRatio.cs" />
+    <Compile Include="DvdTypes\DvdVideoMpegVersion.cs" />
+    <Compile Include="DvdTypes\DvdVideoPermittedDisplayFormat.cs" />
+    <Compile Include="DvdTypes\DvdVideoResolution.cs" />
+    <Compile Include="DvdTypes\DvdVideoStandard.cs" />
+    <Compile Include="Model\AudioProperties.cs" />
+    <Compile Include="Model\SubpictureProperties.cs" />
+    <Compile Include="Model\TitleInfo.cs" />
+    <Compile Include="Model\VideoProperties.cs" />
+    <Compile Include="Model\VmgmInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
-  -->
+  -->
 </Project>
\ No newline at end of file
diff --git TMDbLib/TMDbLib.csproj TMDbLib/TMDbLib.csproj
index cf0f77e..580e574 100644
--- TMDbLib/TMDbLib.csproj
+++ TMDbLib/TMDbLib.csproj
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{C0006F17-212A-474F-983E-7DA2F91E0233}</ProjectGuid>
+    <ProjectGuid>{4997C4E7-8717-45DF-909E-03EFFB48DE9F}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TMDbLib</RootNamespace>
@@ -33,10 +33,6 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="RestSharp, Version=104.2.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\RestSharp.104.2.0\lib\net4\RestSharp.dll</HintPath>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
   </ItemGroup>
@@ -114,6 +110,12 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\RestSharp\RestSharp.Net4\RestSharp.Net4.csproj">
+      <Project>{5ff943a5-260f-4042-b4ce-c4977bad4ebb}</Project>
+      <Name>RestSharp.Net4</Name>
+    </ProjectReference>
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
diff --git TvdbLib/TvdbLib.csproj TvdbLib/TvdbLib.csproj
index f22d87c..c621d52 100644
--- TvdbLib/TvdbLib.csproj
+++ TvdbLib/TvdbLib.csproj
@@ -5,7 +5,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.21022</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{23C34CF0-4E66-4F02-9EDC-83F292C6005C}</ProjectGuid>
+    <ProjectGuid>{B47FEB9E-EB56-45CA-8AFB-99AB9B818F0F}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TvdbLib</RootNamespace>
